integration-tests/tests/src/tests/sync/mixed.ts

```diff
@@ -152,18 +175,18 @@ function describeRoundtrip({
 function describeTypes(flexibleSync: boolean) {
```

Comment:
If you rename it in one place, do it here as well, to avoid the `useFlexibleSync: flexibleSync` below.

```suggestion
function describeTypes(useFlexibleSync: boolean) {
```